Formula/libhttpseverywhere.rb

```diff
@@ -0,0 +1,76 @@
+class Libhttpseverywhere < Formula
+  desc "Bring HTTPSEverywhere to desktop apps"
+  homepage "https://github.com/gnome/libhttpseverywhere"
+  url "https://download.gnome.org/sources/libhttpseverywhere/0.2/libhttpseverywhere-0.2.9.tar.xz"
+  sha256 "1226b2f7cc30db28bd1ccb0a84902e999488f1a99847940cbdab3d2e510be3e1"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "vala" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "json-glib"
+  depends_on "libsoup"
+  depends_on "libgee"
+  depends_on "libarchive"
+
+  def install
+    mkdir "build" do
+      system "meson", "--prefix=#{prefix}", ".."
+      system "ninja"
+      system "ninja", "test"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <httpseverywhere.h>
+
+      int main(int argc, char *argv[]) {
+        GType type = https_everywhere_context_get_type();
+        return 0;
+      }
+    EOS
+    ENV.libxml2
+    gettext = Formula["gettext"]
```

Comment:
I was going to suggest that you could with a `map`, but I didn't notice that a few of the include flags had extra directories on the end. My bad!